src/pages/Home/Components/ExploreToolkit.module.css

```diff
@@ -0,0 +1,76 @@
+.title {
```

Comment:
If you cannot do it please add a todo that we need to change this file into `.styles` file